src/int.rs

```diff
@@ -0,0 +1,167 @@
+//! Stack-allocated big signed integers.
+
+use num_traits::Zero;
+use subtle::{Choice, ConditionallySelectable, ConstantTimeEq};
+
+use crate::{Limb, Uint};
+
+mod add;
+mod cmp;
+mod div;
+mod mul;
+mod neg;
+mod sub;
+
+#[derive(Copy, Clone, Hash, Debug, PartialEq)]
+pub struct Int<const LIMBS: usize> {
```

Comment:
this does not seam to be u2 complement but absolute value + sign, right?
If so, Why not represent Int as enum of Positive(Uint<L>), Negative(Uint<L>)?